src/routes/profile.svelte

```diff
@@ -1,15 +1,122 @@
+<script context="module">
+  import getInitialData from '@/utils/get-initial-data.js';
+  import isoForURL from '@/utils/iso-for-url.js';
+
+  export async function preload(page, session) {
+    let timelineFetchedUntil = new Date();
+    timelineFetchedUntil.setMonth(timelineFetchedUntil.getMonth() - 3);
+    const {
+      account,
+      timeline,
+      statistics,
+      notificationSettings,
+      competences,
+    } = await getInitialData(this, session, new Map([
```

Comment:
Wouldn't it be simpler to just take the object itself and return it?
To add more properties, you can do
```js
return {
    ...initialData,
    timelineFetchedUntil,
};
```
Or simply add it to the object